dropwizard-core/src/main/java/io/dropwizard/server/SimpleServerFactory.java

```diff
@@ -136,4 +132,13 @@ public Server build(Environment environment) {
 
         return server;
     }
+
+    @Override
+    public void configure(Environment environment) {
```

Comment:
since this is the same in both `SimpleServerFactory` and `DefaultServerFactory`, should we move this to the `AbstractServerFactory` instead?